scripts/TestHarness/testers/RavenUtils.py

```diff
@@ -63,7 +64,8 @@ def __lookUpPreferredVersion(name):
              ("h5py",__lookUpPreferredVersion("h5py")),
              ("scipy",__lookUpPreferredVersion("scipy")),
              ("scikit-learn",__lookUpPreferredVersion("sklearn")),
-             ("matplotlib",__lookUpPreferredVersion("matplotlib"))]
+             ("matplotlib",__lookUpPreferredVersion("matplotlib")),
+             ("pyside",__lookUpPreferredVersion("pyside"))]
```

Comment:
Either we need to fix bug where pip install command does 'pyside== '  when there is no preferred version, or just remove it from the __pipList